src/main/java/com/softserve/academy/spaced/repetition/service/RegistrationService.java

```diff
@@ -15,12 +15,11 @@
 
 import java.util.Calendar;
 import java.util.Collections;
-import java.util.HashSet;
-import java.util.Set;
 
 @Service
 public class RegistrationService {
 
+    private static final Integer CARDS_NUMBER = 10;
```

Comment:
The same constant is defined in three different places.

Reply:
Where to introduce that variable so that I could reach it from three different files?

Reply:
It is up to you.